sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -1654,6 +1718,9 @@ def set_standard_blob_tier_blobs(
         :type blobs: str or dict(str, Any) or ~azure.storage.blob.BlobProperties
```

Comment:
I've tried to do a good job of checking this (but GitHub's lack of context is... annoying). But just need to fix docstring here:
`Dict[str, Any]` 
Would probably be a good idea to check in `async`, and just do a quick CTRL+F for `dict(`